src/components/Date/Date.css

```diff
@@ -0,0 +1,23 @@
+.size-small {
+    font-size: 20px;
+}
+
+.size-medium {
+    font-size: 30px;
+}
+
+.size-large {
+    font-size: 40px;
+}
+
+.day-small{
+    font-size: 14px;
+}
+
+.day-medium{
+    font-size: 16px;
+}
+
+.day-large{
+    font-size: 18px;
```

Comment:
These class names are not clear enough. It's confusing. What does `size-small` refer to?

You should have 3 main classes:

1. `.small`
2. `.medium`
3. `.large`

Those 3 classes will be put on the root of the component, meaning the root `div`. No need to use classes specifically for two `p`'s. Use CSS to style the children elements based on the parent class.